tests/end2end/data/hints/simple2.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+
+<!-- target: hello.txt -->
```

Comment:
No need for that comment, for the same reason as [before](https://github.com/The-Compiler/qutebrowser/pull/1842#discussion_r74444550) :wink: